dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -143,27 +199,83 @@ private static void setMetadataKeywords(DepositMetadata depositMetadata, Workflo
 
     }
 
+
+    private static void addUriToRelatedIdentifierList(List<RelatedIdentifier> relatedIdentifierList, String workflowAlias) {
+        RelatedIdentifier aliasRelatedIdentifier = new RelatedIdentifier();
+        aliasRelatedIdentifier.setIdentifier(workflowAlias);
+        aliasRelatedIdentifier.setRelation(RelatedIdentifier.RelationEnum.ISIDENTICALTO);
+        relatedIdentifierList.add(aliasRelatedIdentifier);
+    }
+
+    /**
+     * @param workflow    workflow for which DOI is registered
+     * @param workflowVersion workflow version for which DOI is registered
+     * @param dockstoreGA4GHBaseUrl The baseURL for GA4GH tools endpoint (e.g. "http://localhost:8080/api/api/ga4gh/v2/tools/")
+     * @return TRS URL to workflow (e.g. https://dockstore.org/api/api/ga4gh/v2/tools/%23workflow%2Fgithub.com%2FDataBiosphere
+     * %2Ftopmed-workflows%2FUM_variant_caller_wdl/versions/1.32.0/PLAIN-WDL/descriptor/topmed_freeze3_calling.wdl)
+     */
+    private static String createWorkflowTrsUrl(Workflow workflow, WorkflowVersion workflowVersion, String dockstoreGA4GHBaseUrl) {
```

Comment:
I feel like this deserves a unit test. If it ever needs to be tweaked, it will be hard to ensure that a change doesn't break something.

Reply:
Please write a unit test for this method.

Reply:
done